client/src/partials/templates/modals/printData.modal.js

```diff
@@ -2,32 +2,46 @@ angular.module('bhima.controllers')
 .controller('PrintDataModalController', PrintDataModalController);
 
 // dependencies injection
-PrintDataModalController.$inject = ['SessionService', '$uibModalInstance', 'data'];
+PrintDataModalController.$inject = [
```

Comment:
This is a really useful ,odal!

Since you are displaying a report in this modal, a better name for it would be `ReportModal`.  It seems like a small thing, but you could rename this modal (controller and file) to being the `ReceiptModal`?

I could use it in the `PatientRegistry` to print the patient registration reports once it lands.